spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -196,11 +107,15 @@ While it may seem small, quite a bit is going on. For more about Spring
 Boot specifics, see the
 http://docs.spring.io/spring-boot/docs/current/reference/html/[Spring Boot reference documentation].
 
-We also need to create an `application.properties` file in `src/main/resources`. We need
-to configure two properties in `application.properties`: We need to set the application
-name (which is translated to the task name), and we need to set the logging for Spring
-Cloud Task to `DEBUG` so that we can see what's going on. The following example shows how
-to do both:
+Now let's open the `application.properties` file in `src/main/resources`.
```

Comment:
Change "Now let's" to "Now we can" - still avoiding "let's" (see the earlier comment about it).